.changeset/four-flowers-do.md

```diff
@@ -0,0 +1,19 @@
+---
+'ember-browser-services': patch
+---
+
+Address an issue where the types of each of the browser services did not have sufficient overlap with `Service`.
+
+This is required for for folks using more modern TypeScript in ember apps,
```

Comment:
super nit: extra `for` in `for for` (no need to block PR tho)

Reply:
thanks -- I don't know what happens between my brain and figures when this occurs